app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationURLBar.java

```diff
@@ -135,7 +135,7 @@ else if (aURL.startsWith("resource:") || SessionStore.get().isHomeUri(aURL))
                 aURL = "";
             else if (aURL.startsWith("data:") && SessionStore.get().isCurrentSessionPrivate())
                 aURL = "";
-            else if (aURL.startsWith("javascript:"))
+            else if (aURL.startsWith("javascript:") && SessionStore.get().isHomeUri(aURL))
```

Comment:
not sure how this could work because `aURL` cannot start with `javascript:` and also start with `https://webxr.today/`